telemetry/specs/INT.mdk

```diff
@@ -298,18 +298,64 @@ exhausting header space in the case of a forwarding loop or any other anomalies,
 it is strongly recommended to limit the number of total INT metadata fields
 added by devices.
 
+### MTU Settings
+
 As with any modification that potentially changes the packet size, this
-on-the-fly allocation may “grow” a packet beyond the original MTU, resulting
-in fragmentation. It is advisable to plan network/NIC/MTU settings accordingly
-to leave headroom for the additional headers. 
+on-the-fly allocation may “grow” the packet size, potentially causing it to
+exceed the MTU on an INT switch egress link.
+
+This may be addressed in the following ways -
+
+* It is recommended that the MTU of links between INT sources and sinks is
+configured to be higher than the MTU of preceding link MTUs (server/VM NIC MTUs)
+by an appropriate amount. Configuring an MTU differential of
+(Instruction Count\*4\*INT Hop Count + 8), based on conservative values of
+total number of INT hops and instruction count being used, will prevent egress
+MTU being exceeded due to INT metadata insertion at INT hops.
+
+* An INT source/transit switch may optionally aid in dynamic discovery of Path MTU
+along flows being monitored by INT by transmitting ICMP message as per Path MTU
+Discovery mechanisms of the corresponding IP protocol
+(RFC 1191 for IPv4, RFC 1981 for IPv6). An INT source or transit switch may report
+a conservative MTU in the ICMP message, accounting for metadata insertion at
+downstream INT hops and assuming uniform egress MTU at downstream hops. This
+will help in the path MTU discovery source converging to a path MTU estimate
+faster, although this would be a conservative path MTU estimate.
+Alternatively, each INT hop may report an MTU only accounting for the metadata
+it inserts. This would enable the path MTU discovery source converge to a
+precise path MTU, at the cost of receiving more ICMP messages, one from each
+INT hop.
+
+Regardless of whether or not an INT transit switch participates in Path MTU
+discovery, if it cannot insert all requested metadata because doing so will
+cause the packet length to exceed egress link MTU, it
+**must not insert any metadata and set the M bit** in the
+INT header, indicating that egress MTU was exceeded at an INT hop.
```

Comment:
Though we say below at the M bit description, let's clarify here that the device must not add any metadata to the packet.

Reply:
It is already clarified both here and below in the M bit description.